models/engine/db_storage.py

```diff
@@ -74,3 +74,25 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """A method to retrieve one object
+        """
+        if cls is not None:
+            for key, value in self.all(cls).items():
+                if id in key:
+                    return(value)
+                else:
+                    return (None)
+
+    def count(self, cls=None):
+        """A method to count the number of objects in storage
+        """
+        counter = 0
+        for key, value in self.all().items():
```

Comment:
is not necessary iterate the dict you can use len function
https://www.tutorialspoint.com/python3/dictionary_len.htm